F-FrontEnd/src/fincludes/iso_c_binding.f90

```diff
@@ -0,0 +1,195 @@
+module iso_c_binding
+    implicit none
+
+    ! TODO detect architecture and set correct value
+    integer, parameter :: POINTER_LEN = 4 ! 32 bit
```

Comment:
@h-murai @a-wada-rk It would be nice to be able to set the `POINTER_LEN` value at configuration time depending on the architecture. I was thinking to have a `iso_c_binding.f90.in` file and configure it on the fly when we run configure. How can I achieve that. Is there a `configure` variable that hold the architecture 32|64.